tests/TEST_TYPES

```diff
@@ -101,6 +101,7 @@ Total Spread (Berry) : !6
 NVP |   1  2  !8
 Alpha GW direct gap !9
 Beta GW direct gap !9
+GW+SOC bandgap (eV) !4
```

Comment:
You also need to increment the counter on the first line when adding a new test type.

Reply:
Thanks, Ole!